repository/src/main/java/org/alfresco/repo/rendition2/LocalTransformClient.java

```diff
@@ -145,7 +155,7 @@ public void transform(NodeRef sourceNodeRef, RenditionDefinition2 renditionDefin
                             throw new IllegalStateException("isSupported was not called prior to an asynchronous transform.");
                         }
 
```

Comment:
When using DUAs, we will not be using the contentReader. You may wish to null it out or just ignore it later on. If you do just ignore it, you will need to remove the DIRECT_ACCESS_URL from the options once the first step in a pipeline has been executed.

Will need to do something similar in the LocalSynchronousTransformClient class, but there avoiding creating a reader is harder.

Reply:
`                        
setDirectAccessUrlIfEnabled(actualOptions, sourceNodeRef, localTransform);

localTransform.transform(reader, writer, actualOptions, renditionName, sourceNodeRef);

removeDirectAccessUrlAfterTransform(actualOptions);
` 
line 171, removing DAU after transform request, is it sufficient?